redisgears_v8_plugin/src/lib.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) fn get_exception_msg(
         let trace_utf8 = try_catch.get_trace(ctx_scope).map(|v| v.to_utf8().unwrap());
         GearsApiError::new_verbose(
             error_utf8.as_str(),
-            trace_utf8.as_ref().map(|v| v.as_str().replace("\n", "|")),
+            trace_utf8.as_ref().map(|v| v.as_str().replace('\n', "|")),
```

Comment:
```suggestion
            trace_utf8.as_ref().map(|v| v.as_str().replace('\n',  '|')),
```

Reply:
I guess if we change one we should also change the other? But did not check if replace accept that ..